substrate/primitives/runtime/src/generic/unchecked_extrinsic.rs

```diff
@@ -230,8 +300,11 @@ pub struct UncheckedExtrinsic<Address, Call, Signature, Extension> {
 	/// Information regarding the type of extrinsic this is (inherent or transaction) as well as
 	/// associated extension (`Extension`) data if it's a transaction and a possible signature.
 	pub preamble: Preamble<Address, Signature, Extension>,
-	/// The function that should be called.
-	pub function: Call,
+	/// The function that should be called. It is initially stored in the form that it is provided
+	/// (encoded/decoded) and if needed it will be decoded (and probably also cached)
+	/// the first time it is accessed.
+	#[codec(skip)]
```

Comment:
`#[codec(skip)]` doesn't look so good here. It would look nicer if we manually implement `DecodeWithMemTracking` same as we manually implement `Decode`.

Reply:
If we implement it manually, we could forget to update it when we add a new field. Seemed safer to have it derived automatically. WDYT ?